MPSGameLift/Scripts/install.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Place this script in the GameLift server package
+# GameLift will run this script before opening the server
+# https://docs.o3de.org/docs/user-guide/gems/reference/aws/aws-gamelift/build-packaging-for-linux/
+
+sudo cp -a ./lib64/libX* /lib64/.
+sudo cp -a ./lib64/libbsd* /lib64/.
+sudo cp -a ./lib64/libstdc++* /lib64/.
+
+sudo yum update -y
+sudo yum groupinstall 'Development Tools' -y
+sudo yum install python3 -y
+sudo yum -y install libunwind-devel
+
+echo 'Update outdated make package'
+mkdir make && cd make
+wget http://ftp.gnu.org/gnu/make/make-4.2.1.tar.gz
```

Comment:
is there not a secure `https://` version of this we can use?

Reply:
fixed!